StripeFinancialConnections/StripeFinancialConnections/Source/Native/NativeFlowController.swift

```diff
@@ -539,6 +539,23 @@ extension NativeFlowController: ResetFlowViewControllerDelegate {
     }
 }
 
+// MARK: - NetworkingLinkSignupViewControllerDelegate
+
+@available(iOSApplicationExtension, unavailable)
+extension NativeFlowController: NetworkingLinkSignupViewControllerDelegate {
+
+    func networkingLinkSignupViewControllerDidFinish(
+        _ viewController: NetworkingLinkSignupViewController
+    ) {
+        let successViewController = CreatePaneViewController(
+            pane: .success,
```

Comment:
is this hard coding of next pain a temp thing? Seems like we get manifest back as a result of signup and can use that value?

Reply:
@vardges-stripe Stripe.js does this, but we also call this when a user presses "Not Now" (which we make no network call for)